drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/handlers/KeyboardOperationEditScenarioSimulationGridCell.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.scenariosimulation.client.handlers;
+
+import org.drools.workbench.screens.scenariosimulation.client.metadata.ScenarioHeaderMetaData;
+import org.drools.workbench.screens.scenariosimulation.client.utils.ScenarioSimulationGridHeaderUtilities;
+import org.drools.workbench.screens.scenariosimulation.client.widgets.ScenarioGridColumn;
+import org.uberfire.ext.wires.core.grids.client.model.GridColumn;
+import org.uberfire.ext.wires.core.grids.client.model.GridData;
+import org.uberfire.ext.wires.core.grids.client.widget.context.GridBodyCellEditContext;
+import org.uberfire.ext.wires.core.grids.client.widget.grid.GridWidget;
+import org.uberfire.ext.wires.core.grids.client.widget.grid.impl.KeyboardOperationEditCell;
+import org.uberfire.ext.wires.core.grids.client.widget.grid.renderers.grids.impl.BaseGridRendererHelper;
+import org.uberfire.ext.wires.core.grids.client.widget.layer.GridLayer;
+
+import static org.drools.workbench.screens.scenariosimulation.client.utils.ScenarioSimulationGridHeaderUtilities.isEditableHeader;
+import static org.uberfire.ext.wires.core.grids.client.util.ColumnIndexUtilities.findUiColumnIndex;
+import static org.uberfire.ext.wires.core.grids.client.util.RenderContextUtilities.makeRenderContext;
+
+public class KeyboardOperationEditScenarioSimulationGridCell extends KeyboardOperationEditCell {
+
+    public KeyboardOperationEditScenarioSimulationGridCell(final GridLayer gridLayer) {
+        super(gridLayer);
+    }
+
+    @Override
+    protected void editCell(GridWidget gridWidget) {
```

Comment:
I agree, however not sure what action item are you asking me for. 

I can definitely file some task for me to move logic of headers edition into `appformer` repository to avoid extending `KeyboardOperationEditCell` for `SceSim` and `DMN` separately. Please let me know what do you think.